src/core/public/chrome/ui/header/header.tsx

```diff
@@ -203,6 +204,7 @@ export function Header({
                 toggleCollapsibleNavRef.current.focus();
               }
             }}
+            customNavLink$={observables.customNavLink$}
```

Comment:
So the navlink is only available in `modern` navigation mode?  Is this an expected functional behavior?